voltixApp/VoltixApp/Services/RpcService.swift

```diff
@@ -50,7 +50,7 @@ class RpcService {
             
             if let error = response["error"] as? [String: Any], let message = error["message"] as? String {
                 
-                if message.contains("known") || message.contains("already known") || message.contains("Transaction is temporarily banned") {
+                if message.contains("known") || message.contains("already known") || message.contains("Transaction is temporarily banned") || message.contains("nonce too low: next nonce"){
```

Comment:
`nonce too low`, this one it depends , it could be the transaction already broadcast successfully and committed to block , or it could be we didn't get the latest nonce when we create the transaction

Are you sure we should treat as success?

Reply:
We always get the latest nonce before sending, so I believe this error will only happen if it tries to broadcast the same transaction 2x. The full message is ```nonce too low: next nonce 3 of 4```.